jack-core/src/com/rapleaf/jack/transaction/DbMetricsImpl.java

```diff
@@ -31,19 +29,19 @@
   private static final Logger LOG = LoggerFactory.getLogger(DbMetricsImpl.class);
 
   public DbMetricsImpl(int maxTotalConnections, int minIdleConnections, long maxWaitMillis, long keepAliveMillis) {
-    this.lifeTimeStopwatch = new Stopwatch();
-    this.lifeTimeStopwatch.start();
+    startTime = System.currentTimeMillis();
```

Comment:
Since all the other parameters here are referred to with `this.`, let's keep the same style for `startTime`.